webapp/Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: help
+.DEFAULT_GOAL := help
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+setup-env:					## Setup the environment
+	cp .env.dist .env
+
+install: setup-env			## Install dependencies
+	composer install
```

Comment:
Also, this requires a local composer installation to work, but this is not documented.
Since this project is mostly intended to be run with Docker, maybe you can simply check if composer is installed locally, and do nothing if it ain't.